geode-core/build.gradle

```diff
@@ -366,6 +366,7 @@ dependencies {
 
     integrationTestRuntimeOnly('org.apache.derby:derby')
     integrationTestRuntimeOnly('xerces:xercesImpl')
+    integrationTestRuntimeOnly('com.fasterxml.jackson.datatype:jackson-datatype-jsr310')
```

Comment:
this seems like the wrong scope.  don't we need jackson-datatype-jdr310 jar to be included in the shipping product, not just available to this one test?